Can't output from kinesis-balanced or kinesis

I have the following config and benthos 3.11.0:

`input:
  kinesis_balanced:
    stream: what-a-great-stream
    start_from_oldest: false
    dynamodb_table: table-for-benthos
    dynamodb_billing_mode: PAY_PER_REQUEST
    region: us-east-1
    credentials:
        profile: what-a-great-profile
output:
    file:
        path: outfile.txt
        delimiter: ""
`

Using the same AWS profile, on us-east-1, I'm able to put records on this stream, get records from it, and when I scan dynamodb `table-for-benthos` I see leases are being updated by benthos as long as it's running. The file, `outfile.txt` is getting created, and if I swap the input to stdin it works.

I'm not getting any errors, and I'm not getting any output from kinesis.

I'm so close to living the dream, I can taste it.

No new data sire:
```{"@timestamp":"2020-03-19T19:27:46-07:00","@service":"benthos","level":"INFO","component":"benthos","message":"Launching a benthos instance, use CTRL+C to close."}
{"@timestamp":"2020-03-19T19:27:46-07:00","@service":"benthos","level":"INFO","component":"benthos","message":"Listening for HTTP requests at: http://0.0.0.0:4195"}
{"@timestamp":"2020-03-19T19:27:47-07:00","@service":"benthos","level":"INFO","component":"benthos.input","message":"Receiving Amazon Kinesis messages from stream: respondent-event-stream-dev"}
^C{"@timestamp":"2020-03-19T19:40:15-07:00","@service":"benthos","level":"INFO","component":"benthos","message":"Received SIGTERM, the service is closing."}
{"@timestamp":"2020-03-19T19:40:15-07:00","@service":"benthos","level":"DEBUG","component":"benthos.input","message":"Waiting for pending acks to resolve before shutting down."}
{"@timestamp":"2020-03-19T19:40:15-07:00","@service":"benthos","level":"DEBUG","component":"benthos.input","message":"Pending acks resolved."}
panic: close of closed channel

goroutine 330 [running]:
github.com/patrobinson/gokini.(*KinesisConsumer).Shutdown(0xc000230c00)
	/home/ash/go/pkg/mod/github.com/patrobinson/gokini@v0.0.7/consumer.go:249 +0x37
created by github.com/Jeffail/benthos/v3/lib/input/reader.(*KinesisBalanced).CloseAsync
	/home/ash/src/go/benthos/lib/input/reader/kinesis_balanced.go:207 +0x46
```
With config:

```
input:
  kinesis_balanced:
    stream: what-a-great-stream
    start_from_oldest: true
    dynamodb_table: table-for-benthos
    dynamodb_billing_mode: PAY_PER_REQUEST
    region: us-east-1
    credentials:
        profile: what-a-great-profile

output:
  file:
    path: outfile.txt
    delimiter: ""

logger:
  level: TRACE

metrics:
  type: stdout```

--- a/go.mod
+++ b/go.mod
@@ -47,6 +47,8 @@ require (
 	github.com/linkedin/goavro/v2 v2.9.7
 	github.com/mailru/easyjson v0.7.0 // indirect
 	github.com/microcosm-cc/bluemonday v1.0.2
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/nats-io/jwt v0.3.2 // indirect
 	github.com/nats-io/nats-streaming-server v0.16.1-0.20190905144423-ed7405a40a25 // indirect
 	github.com/nats-io/nats.go v1.9.1
@@ -60,10 +62,9 @@ require (
 	github.com/opencontainers/runc v0.1.1 // indirect
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/ory/dockertest v3.3.4+incompatible
-	github.com/patrobinson/gokini v0.0.7
+	github.com/patrobinson/gokini v0.1.0
 	github.com/pebbe/zmq4 v1.0.0
 	github.com/pierrec/lz4 v2.4.0+incompatible // indirect
-	github.com/pkg/errors v0.9.0 // indirect
 	github.com/prometheus/client_golang v1.3.0
 	github.com/prometheus/common v0.8.0 // indirect
 	github.com/quipo/dependencysolver v0.0.0-20170801134659-2b009cb4ddcc

